packages/components/src/internal/components/search/actions.ts

```diff
@@ -210,7 +210,7 @@ export function resolveIconSrc(data: any, category: SearchCategory): string {
             // else fallback to default
         } else if (data.sampleSet?.name) {
             iconSrc = 'samples';
-        } else if (data.type) {
+        } else if (data.type && category !== SearchCategory.PlateSet) {
```

Comment:
Slightly hacky... currently without this conditional the 'type' on a PS makes logic flow into this block

Reply:
I'm not seeing why going into this block is a problem. Does the `lcType` for PS end up matching on one of these conditions? I'd assume it would just fall through.

Reply:
It falls through and renders like so: 
![Screenshot 2024-11-22 at 11 34 44 AM](https://github.com/user-attachments/assets/4d11322a-0f11-434e-976f-7dd2a4076f9a)